applications/asset_tracker_v2/sample.yaml

```diff
@@ -364,3 +364,18 @@ tests:
                 EXTRA_CONF_FILE="overlay-nrf7002ek-wifi-scan-only.conf;overlay-debug.conf"
                 SB_CONFIG_WIFI_NRF700X=y SB_CONFIG_WIFI_NRF700X_SCAN_ONLY=y
     tags: ci_build sysbuild
+
```

Comment:
Can't the CI be modified instead? That would be cleaner I believe

Reply:
We have CI compiling these configurations at the moment so everything is working but it would be more efficient from integration testing perspective if we could use the images from twister stage.
We are already doing that with MOSH tests. This PR targets for us to be able to do that also for positioning test set, which is using ATv2, GNSS sample and Location sample.

Reply:
I see. I believe it is better to call this new configuration `locationing-ext-gnss-antenna-debug` . This will make it similar to other configurations. That way we do not unnecessarily expose details about our CI here. You can remove the comment in line 368 as this may be a useful configuration to have for customers as well. 

@jorgenmk  Will adding this new configuration have any effect on our CI plan?

Reply:
What are the potential secret details of the CI this is revealing? I'm asking it because MOSH has it already. And I don't really see any problems sharing information that we are testing with this configuration.

There is a benefit on the naming and comment indicating that this configuration is used for testing purposes. People don't break it as easily and are more careful in modifying it.

I don't see how this would affect on other CI plans than the ones that intend to use it.

Reply:
I agree that there are no secrets being revealed. But it just makes the sample.yaml clumsy. I still prefer that you that change the name to keep it more generic. But I wont insist further. Will approve.

Reply:
I agree that having those integration_config's doesn't look nice.
However, I find it even worse if we add a somewhat weird config there without any reasoning. It will cause more confusion, discussion and questioning why it's there. So instead of keeping the reasoning part secret, writing it down clearly is better in my opinion.

Reply:
Got it. Go for it 🙂